internal/host/host.go

```diff
@@ -306,6 +309,10 @@ func (m *Manager) updateInventory(ctx context.Context, cluster *common.Cluster,
 		return common.NewApiError(http.StatusNotFound, err)
 	}
 
+	if m.autorenameHost(h, inventory) {
```

Comment:
So are we officially giving up on delayed rDNS? Can you please state this in the commit message and a code comment?

Reply:
Yes @omertuc, that is one of the side effects of not adding an additional field: there is no way to know if the name in "requested_hostname" came from the user or was automatically calculated from a MAC address. So either we allow host name discovered via reverse DNS after the initial discovery to override the hostname explicitly requested by the user or else we ignore it. I will explain that.

Reply:
Also it would be nice to investigate whether my hypothetical scenario is something that happens often in prod or not, before we go through with this